src/acceptance-test/java/org/zalando/nakadi/webservice/CursorsServiceAT.java

```diff
@@ -173,14 +156,14 @@ public void whenETRepoExceptionThenException() throws Exception {
 
     @Test
     public void whenFirstCursorIsNotCommittedThenNextCursorsAreNotSkipped() throws Exception {
-        final SubscriptionCursor c1 = new SubscriptionCursor(P1, OLDEST_OFFSET, etName, token);
-        final SubscriptionCursor c2 = new SubscriptionCursor(P2, NEW_OFFSET, etName, token);
+        final NakadiCursor c1 = new NakadiCursor(timeline, P1, OLDEST_OFFSET);
+        final NakadiCursor c2 = new NakadiCursor(timeline, P2, NEW_OFFSET);
         testCursors = ImmutableList.of(c1, c2);
 
-        final List<CursorCommitResult> result = cursorsService.commitCursors(streamId, sid, testCursors);
+        final List<Boolean> result = cursorsService.commitCursors(streamId, sid, testCursors);
 
-        assertThat(result.get(0).getResult(), equalTo(CursorCommitResult.OUTDATED));
-        assertThat(result.get(1).getResult(), equalTo(CursorCommitResult.COMMITTED));
+        assertEquals(Boolean.FALSE, result.get(0));
+        assertEquals(Boolean.TRUE, result.get(1));
```

Comment:
if you don't use hamcrest-style anyway, you can just use here assertTrue and assertFalse